inc/class-upgrade.php

```diff
@@ -1161,4 +1164,77 @@ public function migrate_og_settings_from_social_to_titles() {
 		\update_option( 'wpseo_social', $wpseo_social );
 		\update_option( 'wpseo_titles', $wpseo_titles );
 	}
+
+	/**
+	 * Migrates the frontpage social settings to the titles options.
+	 *
+	 * @return void
+	 */
+	public function set_og_settings_from_seo_values() {
+		$wpseo_titles    = \get_option( 'wpseo_titles' );
```

Comment:
We're a bit inconsistent in the usage of the backslack for global namespace, but I think the conclusion was to not use it in non-namespaced files?

Reply:
Right, will fix